fava/templates/_layout.html

```diff
@@ -36,23 +37,16 @@
         </svg>
-        <h1 class="site-name">{{ api.title or 'fava' }}{{ ' ▾' if config['APIS']|length > 1 else '' }}<strong>{{ page_title }}</strong>
-            {% if config['APIS']|length > 1 %}
-            <div class="beancount-files">
-                <ul>
-                {% for file_slug, file_api in config['APIS'].items() %}
-                    <li{% if file_slug == g.beancount_file_slug %} class="selected"{% endif %}>
-                    <a href="{{ url_for('index', bfile=file_slug) }}">{{ file_api.title or 'fava' }}</a>
-                    </li>
-                {% endfor %}
-                </ul>
-        </div>{% endif %}</h1>
+        <h1 class="site-name">{{ api.title or 'fava' }}{{ ' ▾' if config['APIS']|length > 1 else '' }}<strong>{{ page_title }}</strong></h1>
         {% if show_filters %}
         <nav>
             {% include "_entry_filters.html" %}
         </nav>
         {% endif %}
     </header>
+    <a href="#" id="aside_button">
+        <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" enable-background="new 0 0 24 24" id="Layer_1" version="1.0" viewBox="0 0 24 24" xml:space="preserve"><circle cx="12" cy="12" r="2"/><circle cx="12" cy="5" r="2"/><circle cx="12" cy="19" r="2"/></svg>
```

Comment:
Add some width/height to the svg so that this doesn't turn out to be some freakishly huge dot if the CSS doesn't load.